pkg/systemlogmonitor/log_monitor.go

```diff
@@ -129,7 +129,11 @@ func (l *logMonitor) monitorLoop() {
 	l.initializeStatus()
 	for {
 		select {
-		case log := <-l.logCh:
+		case log, ok := <-l.logCh:
+			if !ok {
+				glog.Errorf("Log channel closed")
```

Comment:
We should add more information in the log.

I'll send another change to address this. I want to fix something else anyway.